src/Form.php

```diff
@@ -532,13 +532,12 @@ protected function renderTemplateToHtml(string $region = null): string
     {
         $output = parent::renderTemplateToHtml($region);
 
-        return $this->fixFormInRenderedHtml($output);
+        return $this->fixOwningFormAttrInRenderedHtml($output);
     }
 
-    public function fixFormInRenderedHtml(string $html): string
+    public function fixOwningFormAttrInRenderedHtml(string $html): string
     {
-        $innerFormTags = ['button', 'datalist', 'fieldset', 'input', 'keygen', 'label', 'legend',
-            'meter', 'optgroup', 'option', 'output', 'progress', 'select', 'textarea', ];
+        $innerFormTags = ['button', 'fieldset', 'input', 'output', 'select', 'textarea'];
```

Comment:
all other removed tags were added wrongly, they have no `form` nor `for` attribute

`label` is non-value tag /w `for` attribute, removing now, hopefully not needed - https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label